source/docs/software/advanced-controls/filters/slew-rate-limiter.rst

```diff
@@ -52,7 +52,7 @@ Once your filter has been created, using it is easy - simply call the ``calculat
   .. code-block:: c++
 
     // Calculates the next value of the output
-    filter.Calculate(input);
+    filter.Calculate(units::voltage::volt_t{input}).value();
```

Comment:
This doesn't make sense, why are you creating a unit then calling .value on the newly created unit?

Reply:
Please see the new comment for clarification.

Reply:
The new comment doesnt address this the whole change here should be removed from what I've seen but this definitely isn't correct